inference-engine/tests/functional/plugin/shared/include/behavior/config.hpp

```diff
@@ -72,11 +72,15 @@ namespace BehaviorTestsDefinitions {
         SKIP_IF_CURRENT_TEST_IS_DISABLED()
         // Create CNNNetwork from ngrpah::Function
         InferenceEngine::CNNNetwork cnnNet(function);
-        if (targetDevice.find(CommonTestUtils::DEVICE_AUTO) != std::string::npos) {
-            GTEST_SKIP();
+        if (targetDevice.find(CommonTestUtils::DEVICE_AUTO) == std::string::npos) {
+            auto execNet = ie->LoadNetwork(cnnNet, targetDevice, configuration);
         } else {
-            ASSERT_THROW(auto execNet = ie->LoadNetwork(cnnNet, targetDevice, configuration),
-                         InferenceEngine::Exception);
+            // "PERF_COUNT" is accepted by auto-device plugin
+            if (configuration.find("PERF_COUNT") == configuration.end()) {
```

Comment:
end() points to the past-the-end element in the map :) 
also being in the end != that conf is ONLY PERF_COUNT